src/rebar_coveralls.erl

```diff
@@ -91,11 +106,25 @@ eunit_test_() ->
   Get            = fun(cover_export_enabled, _) -> true end,
   GetLocal       = fun(coveralls_coverdata, _)      -> File;
                       (coveralls_service_name, _)   -> ServiceName;
-                      (coveralls_service_job_id, _) -> ServiceJobId
+                      (coveralls_service_job_id, _) -> ServiceJobId;
+                      (do_coveralls_after_eunit, _) -> true;
```

Comment:
it would be better if these two parameter were not specified, at it is the most likely usage of the software